.circleci/config.yml

```diff
@@ -45,10 +45,17 @@ aliases:
           command: yarn lint
       - run:
           name: Test
-          command: yarn test-ci
+          command: yarn nyc --clean yarn test-ci --coverage
```

Comment:
Actually, I don't think we need to run nyc for unit tests since they don't (or shouldn't) launch any sub processes.